00-HTML-CSS-basics/layout2.6_js.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout2.6_js.css" rel="stylesheet" type="text/css">
+<!-- THIS IS JUST TO FIX THE BUG I HAD WITH THE COLOR OF THE IMAGES WHEN THE MOUSE POINTER WAS OVER THE TEXT -->    
+    <script type="text/javascript">
+        function color1() {
+            document.getElementById("i1").style.filter = 'grayscale(0%)';
+        }
+
+        function color2() {
+            document.getElementById("i2").style.filter = 'grayscale(0%)';
+        }
+
+        function color3() {
+            document.getElementById("i3").style.filter = 'grayscale(0%)';
+        }
+
+        function color4() {
+            document.getElementById("i4").style.filter = 'grayscale(0%)';
+        }
+
+        function color5() {
+            document.getElementById("i5").style.filter = 'grayscale(0%)';
+        }
+
+        function color6() {
+            document.getElementById("i6").style.filter = 'grayscale(0%)';
+        }
+
+        function nocolor1() {
+            document.getElementById("i1").style.filter = 'grayscale(100%)';
+        }
+        function nocolor2() {
+            document.getElementById("i2").style.filter = 'grayscale(100%)';
+        }
+        function nocolor3() {
+            document.getElementById("i3").style.filter = 'grayscale(100%)';
+        }
+        function nocolor4() {
+            document.getElementById("i4").style.filter = 'grayscale(100%)';
+        }
+        function nocolor5() {
+            document.getElementById("i5").style.filter = 'grayscale(100%)';
+        }
+        function nocolor6() {
+            document.getElementById("i6").style.filter = 'grayscale(100%)';
+        }
+
+    </script>
+    <title>Layout</title>
+</head>
+<body>
+
+<div class="container">
+    <div class="header">
+        <p class="showcase">Showcase</p>
+        <aside>
+            <p>
+                Incredible Types is a curated collection and showcase of outstanding typography and desing from around 
+                the world. We currently have 354 pieces of inspirational desing in our showcase, from 365 creatives 
+                and studios from 46 different countries.
+            </p>
+            <p>
+                Follow us on <span class="link"><a href="#">Twitter</a></span>, <span class="link"><a href="#">Facebook</a></span> and
+                <span class="link"><a href="#">Pinterest</a></span> or suscribe to our <span class="link"><a href="#">RSS Feed</a></span>
+            </p>
+        </aside>
+    </div>
+    <div class="images">
+        <div class="col">
+            <div class="img top">
+                <a href="#"><img id="i1" onmouseover="color1()" onmouseout="nocolor1()" src="img1.jpg"></a>
+                <span class="tooltip" onmouseover="color1()" onmouseout="nocolor1()"><a class="tit" href="#">Title</a><br><a class="des" href="#">description</a></span>
```

Comment:
You're using a break line here but in code you're not creating a new line. That's hard to follow.